test/v4/kyberUniswapReserve.js

```diff
@@ -715,12 +711,12 @@ contract("KyberUniswapReserve", async accounts => {
                 1 /* eth */,
                 1 /* token */
             );
-            const amount = web3.toWei("1");
-            const conversionRate = new BigNumber(10).pow(18);
+            const amount = web3.utils.toWei("1");
+            const conversionRate = new BN(10).pow(new BN(18));
```

Comment:
for next files.
can do
new BN(10 ** 18)
also other combinations can  be shorter like this.